cloudbuild.yaml

```
@@ -1,109 +1,12 @@
+# Modern Cloud Build configuration
+# Optimized for containerization and deployment
+# Integrates with GitHub Actions CI/CD pipeline
+
+substitutions:
+  _ARTIFACT_REGISTRY: 'asia-southeast1-docker.pkg.dev'
+  _GCP_REGION: 'asia-southeast1'
+  _DULCE_AGENTS_TOPIC: 'dulce.agents'
+  _DULCE_AGENT_RUNS_TABLE: 'dulce_agent_runs'
+
+steps:
+  #
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)